iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/InstructionWithExecOrShellFormTreeTree.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.docker.tree.api;
+
+
+import java.util.List;
+
+public interface InstructionWithExecOrShellFormTreeTree extends InstructionTree {
+
+  ExecFormTree execForm();
+
+  ShellFormTree shellForm();
```

Comment:
I think we don't need these getters for now. We are more interested in the arguments itself.